en/02-javascript/0202-modelo-de-datos/index.md

```diff
@@ -438,7 +438,7 @@ enemy; // notice the position again.
 The value of `this` is one of JavaScript's most debated aspects.
 
 In other languages, methods and functions are different things and a method
-_always_ has one -and only one- associated object, so `this` never changes.

```

Comment:
Those hyphens come from the original. I considered using the dashes instead, but in the end I was too afraid of going Against The Author. I think it just looks better with dashes, so approved.